apps/server/src/modules/files-storage/uc/files-storage.uc.spec.ts

```diff
@@ -184,6 +191,24 @@ describe('FilesStorageUC', () => {
 			});
 		});
 
+		describe('Tests of permission handling', () => {
+			it('should call authorizationService.hasPermissionByReferences', async () => {
+				authorizationService.hasPermissionByReferences.mockResolvedValue(true);
+				await service.upload(userId, fileUploadParams, request);
+				expect(authorizationService.hasPermissionByReferences).toBeCalledWith(
+					userId,
+					fileUploadParams.parentType,
+					fileUploadParams.parentId,
+					{ action: Actions.read, requiredPermissions: ['FILE_CREATE'] }
```

Comment:
Action.write it make no different but is what we have. Maybe we should check if the right params are passed.